Lib/ntpath.py

```diff
@@ -93,7 +93,7 @@ def isabs(s):
         double_sep = '\\\\'
     s = s[:3].replace(altsep, sep)
     # Absolute: UNC, device, and paths with a drive and root.
-    return s.startswith(colon_sep, 1) or s.startswith(double_sep)
+    return (s[1:3] == colon_sep and s[:1] != sep) or s[:2] == double_sep
```

Comment:
How much faster is the version that uses slices instead of `startswith()`?

Reply:
The fixed version that uses slices is 1.14x faster than the one with `startswith()`.
The fixed version that uses `startswith()` would be 1.24x slower than the old one.
See the benchmark at the top.